### PR TITLE
Change takeover image on mobile

### LINKED ISSUE #8509: Improvement suggestion for homepage on small screens
When you view the homepage on your phone the image of the takeover takes up a lot of space.

Screenshot_20201017-101215

I would suggest on small screens we use the [stacked heading icon](https://vanillaframework.io/docs/patterns/heading-icon#stacked).

Screenshot_20201017-101415

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Change image header on mobile for some of the takeovers. 

## QA
- Go to https://ubuntu-com-8520.demos.haus
- Look at the takeover does the image header look good on large, medium and small? Some of the take over use the stacked image design: https://vanillaframework.io/docs/patterns/heading-icon#stacked

## Issue / Card

Fixes #8509 

## Screenshots

image

### CODE REVIEW

File: templates/takeovers/_template.html [hunk 1]
```diff
@@ -1,68 +1,75 @@
...
-          {{ title | safe }}
-        </h1>
-        {% if subtitle %}<h4>
-          {{ subtitle | safe }}
-        </h4>{% endif %}
-        <div class="u-hide--small">
-          {% if primary_url %}
+      {% if stacked_image %}
+        <div class="u-hide u-show--small">
+          <img class="p-heading-icon__img " src="{{ header_image }}" alt="">
```
Comment (anthonydillon) [inline templates/takeovers/_template.html:6]: We should use the full markup for the heading icon pattern. Including the heading. By using this class outside of this block pattern leads to untested and unexpected styling.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks much better @albertkol. I have left a comment in the code and I believe design will request the icon is slightly larger.

File: templates/takeovers/_template.html (hunk 1 above)
Comment (albertkol) [reply templates/takeovers/_template.html]: Made a change. Let me know if this is not what you meant. :D

**Comment by albertkol** [comment]
I will wait for the design feedback.

I do think the icon could be bigger. I am mildly infuriated that Vanilla is only providing me the style for `p-heading-icon--small`. What would be the right approach to make the icon bigger, @anthonydillon? I'd love to be able to do  `p-heading-icon--medium` or `p-heading-icon--large`.

Besides that I think the margin-bottom between elements (icon, h1, h4 and button) seem inconsistent. We should look at all takeovers and see if we can bring them in line. But that's just my opinion.

**Review by anthonydillon — APPROVED** [review]
Code wise LGTM but I think you may want to build a `p-heading-icon--large` extension to the pattern locally which sets the width and height of the icon to `4rem` (twice the size). 

To do that please reference to the Vanilla implementation and try and match that as closely as possible. With a comment above to the issue on Vanilla to propose the pattern upstream.

**Comment by SirSamTumless** [comment]
@spencerbygraves Could you give this a quick review?

**Comment by spencerbygraves** [comment]
@albertkol I agree the icon is too small. Happy to go with @anthonydillon's suggestion above. Thanks.

**Comment by carkod** [comment]
We should QA this change in my takeovers PR #8414. Once I rebase

**Comment by anthonydillon** [comment]
Screenshots:
image

image

**Comment by anthonydillon** [comment]
Looks great @albertkol thanks :+1:

**Comment by SirSamTumless** [comment]
@spencerbygraves Please give this another reivew

**Comment by lyubomir-popov** [comment]
I would propose the following:
48px (3rem) icon on desktop, 40px (2.5rem) icon on smaller screens to match the smaller heading size at those resolutions.


image

**Pull request merged**